AusTraits_tutorial.qmd

```diff
@@ -164,8 +148,32 @@ The columns include:
     -   taxon_name 
     -   trait_name 
     -   value (trait value) 
--   entity & value metadata
+-   entity metadata
+    -   entity_type\
```

Comment:
What do the backslashes do?

Reply:
They are a forced line breaks  and also "escape" codes added before certain characters - and in some circumstances they automatically add themselves